### ISSUE TITLE
Message Updation in Fraction Activity

### ISSUE THREAD

**Issue by Vishalk91-4** [llaske/sugarizer#1453]
![Screenshot 2024-01-03 141754](https://github.com/llaske/sugarizer/assets/116670999/da7bfa48-1c57-40d3-9fdc-f1351db22ff0)

After the game over message, when we click on play and the game starts, 
Then Pausing it befor the first turn, results in showing Game over again instead we could write, Bouncing ball and the fraction on the first turn

@llaske, your views

**Comment by llaske** [comment]
Hmmm. I've reproduce it. Not seems to be a problem.

**Comment by Vishalk91-4** [comment]
> Hmmm. I've reproduce it. Not seems to be a problem.

I was just telling when we first get to play, after one game gets over it should not show "Game over Press play" message

**Comment by llaske** [comment]
Yes I got it. My point is that it's a very minor issue. I wonder if someone could see it.

**Comment by llaske** [comment]
Fixed in https://github.com/llaske/sugarizer/pull/1580
